pkg/webservice/handlers.go

```diff
@@ -619,6 +628,58 @@ func getQueueApplications(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func getPartitionApplicationsByState(w http.ResponseWriter, r *http.Request) {
```

Comment:
Function name: you're not returning applications but application IDs. A better name is `getPartitionApplicationIDsByState()`.